common/util.py

```diff
@@ -1,9 +1,5 @@
-import pytest
 import shutil
 
 
-def skip_unless_irun_available(fn):
-    return pytest.mark.skipif(
-        shutil.which("irun") is None,
-        reason="irun (simulator command) not available"
-    )(fn)
+irun_available = shutil.which("irun") is not None
```

Comment:
Can we make these functions? perhaps me over analyzing, but think it makes sense to check at runtime rather than import time.

We can also add a function `verilog_compiler_available()` which takes the `or` of the underlying conditions.